content/blog/2020/01/2020-01-27-web-socket.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: 'WebSocket'
+tags:
+- core
+- remoting
+- agents
+- cli
+author: jglick
+---
+
+I am happy to report that jep:222[] has landed in Jenkins weeklies,
+starting in link:/changelog/#v2.217[2.217].
+This improvement brings experimental WebSocket support to Jenkins,
+available when connecting inbound agents or when running the CLI.
+The WebSocket protocol allows bidirectional, streaming communication over an HTTP(S) port.
+
+While many users of Jenkins could benefit,
+implementing this system was particularly urgent for CloudBees
+because of how CloudBees Core on modern cloud platforms (i.e., Kubernetes) configures networking.
```

Comment:
Would be nice to clarify what is this product about (short summary in brackets + hyperlink)

Reply:
agreed